src/lightning/app/core/constants.py

```diff
@@ -89,6 +89,11 @@ def get_lightning_cloud_url() -> str:
 ENABLE_STATE_WEBSOCKET = bool(int(os.getenv("ENABLE_STATE_WEBSOCKET", "1")))
 ENABLE_UPLOAD_ENDPOINT = bool(int(os.getenv("ENABLE_UPLOAD_ENDPOINT", "1")))
 
+# directory where system customization sync files stored
+SYS_CUSTOMIZATIONS_SYNC_ROOT = "/tmp/sys-customizations-sync"
```

Comment:
The path should probably be concatenated using `os.path.join` or pathlib, to be OS agnostic.
The location of the temp dir should also be chosen based on the os. You could use the tempfile module for that.

Reply:
This is a CloudSpace-specific thing, and would be run only within plugin-server container with a fixed mount with these files (there is a corresponding logic from BE side)